code/modules/events/space_dragon.dm

```diff
@@ -6,14 +6,15 @@
 	earliest_start = 70 MINUTES
 	min_players = 20
 
+/datum/round_event/space_dragon
```

Comment:
What's this for?

Reply:
It used to have stuff under it, but it was removed.  Good catch, removing now.